### ISSUE TITLE
A11y: length range slider lacks visible focus indication and accessible name

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/browser#2477]
### Steps To Reproduce

For focus indication:

1. Navigating with the keyboard, go to "Generator"
2. Set focus (using `Tab`/`Shift+Tab`) to the Length range slider

---

For accessible name:

1. Run a screenreader (e.g. NVDA on Windows)
2.  go to "Generator"
3. Set focus (using `Tab`/`Shift+Tab`) to the Length range slider

### Expected Result

For focus indication:

- expect to see a visible focus ring or indication on the range slider when it has focus

---

For accessible name:

- expect the slider to be announced with a specific label/name

### Actual Result

For focus indication:

- the range slider lacks any visible focus indication

---

For accessible name:

- the slider is simply announced as an unlabelled/generic "slider"

### Screenshots or Videos

Recording using Chrome and NVDA on Windows, with Speech Viewer enabled. Note how the number/spinbox is announced correctly as "Length spin button ...", while the range slider itself is simply announced as an unlabelled "slider". Note also the lack of visible focus indication when the slider itself receives focus.

https://user-images.githubusercontent.com/895831/160377081-bb981bb2-456d-47f6-a72b-c25553357966.mp4

### Additional Context

I would actually suggest that the range slider should not be keyboard-focusable at all, as it is redundant for a keyboard user (since they can operate the preceding spin box/number input). This neatly sidesteps the issue of lack of visible focus indication.

For screen reader users though, I'd still suggest adding an explicit accessible name to the non-focusable range slider (thinking for instance of situations where a user may be partially sighted, use a mouse, but also a screen reader ... they'll want to hear the accessible name of the control once they're operating it/it's under their mouse). A simple `aria-label`, replicating the "Length" label, should be sufficient.

This is a clone issue of https://github.com/bitwarden/desktop/issues/1458

### Operating System

Windows

### Operating System Version

10

### Web Browser

Chrome

### Browser Version

99.0.4844.84 (Official Build) (64-bit)

### Build Version

1.57.0
